tasks/19_array-chunks/array-chunks.js

```diff
@@ -0,0 +1,7 @@
+export function arrayChunks(arr, n) {
+  const temp = [];
+  for (let i=0; i< arr.length; i+=n) {
```

Comment:
It should be `i < arr` or `i<arr`. I would go with Keep it consistent

Not something that you should be taking care of, but I leave it here as reminder, it would be good to have some eslint rules turned on like in this case it would be `space-infix-ops`

Reply:
I fixed the inconsistencies and will try to be more consistent in the future